### PR TITLE
Update transaction component

### PR DESCRIPTION
**Pull request by xdamman**
## Collective page > Budget section

Closes opencollective/opencollective#1289

### CODE REVIEW

File: src/apps/expenses/components/Expenses.js [hunk 1]
```diff
@@ -94,6 +94,9 @@ class Expenses extends React.Component {
           .itemsList {
             position: relative;
           }
+          .itemsList :global(.item) {
```
Comment (HipsterBrown) [inline src/apps/expenses/components/Expenses.js]: This doesn't need to be `:global` if the className is created in this component.

File: src/apps/expenses/components/ExpensesSection.js [hunk 2]
```diff
@@ -77,48 +92,56 @@ class ExpensesSection extends React.Component {
...
+                <div id="expenses" className="first col">
+                  <div className="header">
+                    <h2>
+                      <FormattedMessage
+                        id="collective.expenses.title"
+                        values={{ n: this.totalExpenses }}
+                        defaultMessage="{n, plural, one {Latest expense} other {Latest expenses}}"
+                        />
+                    </h2>
+                    <SmallButton
```
Comment (HipsterBrown) [inline src/apps/expenses/components/ExpensesSection.js]: Could this be a styled anchor tag somehow? I like having real links for browser accessibility, i.e. opening in a new tab.

File: src/apps/expenses/components/ExpensesSection.js [hunk 3]
```diff
@@ -77,48 +92,56 @@ class ExpensesSection extends React.Component {
...
+                      disabled={this.totalExpenses < 5}
```
Comment (HipsterBrown) [inline src/apps/expenses/components/ExpensesSection.js]: Why show this button at all if we have less than 5 expenses?

**Comment by znarf** [comment]
Some remarks on the PR structuring:
  - No need to have the `Fix/1289` in the PR title.
  - Would be great to have "Closes opencollective/opencollective#1289" in the ticket description.

**Comment by znarf** [comment]
@xdamman have you somehow addressed the feedback from @HipsterBrown in this PR?

File: src/apps/expenses/components/Expenses.js (hunk 1 above)
Comment (xdamman) [reply src/apps/expenses/components/Expenses.js]: good point

**Pull request closed without merging**